examples/actix_example/src/main.rs

```diff
@@ -0,0 +1,234 @@
+// use actix_web::{get, post, web, App, HttpResponse, HttpServer, Responder};
+use std::sync::Arc;
+
+use actix_files as fs;
+use actix_web::{
+    error, get, middleware, post, web, App, Error, HttpRequest, HttpResponse,
+    HttpServer, Result,
+};
+use listenfd::ListenFd;
+use sea_orm::entity::*;
+use sea_orm::{DatabaseConnection, EntityTrait};
+use serde::{Deserialize, Serialize};
+use std::env;
+use tera::Tera;
+
+mod post;
+pub use post::Entity as Post;
+mod setup;
+
+const DEFAULT_POSTS_PER_PAGE: usize = 25;
+
+#[derive(Debug, Clone)]
+struct AppState {
+    templates: tera::Tera,
+    conn: Arc<DatabaseConnection>,
+}
+type SharedState = Arc<AppState>;
```

Comment:
The `conn: Arc<DatabaseConnection>` does not need to be wrapped in an Arc as sqlx's connection struct is designed to be cloned already.

Perhaps this can be updated to this?
```rust
struct AppState {
    templates: Arc<tera::Tera>,
    conn: DatabaseConnection,
}
```

Reply:
Great! Thanks for the input everyone, looks like with the latest updates we don't need Arc at all.